aptos-move/framework/supra-framework/sources/reconfiguration.spec.move

```diff
@@ -113,7 +113,8 @@ spec supra_framework::reconfiguration {
     /// Should equal to 0
     spec emit_genesis_reconfiguration_event {
         use supra_framework::reconfiguration::{Configuration};
-
+        // TODO remove aborts_if_is_partial after the property proved
+        pragma aborts_if_is_partial;
```

Comment:
`AbortsIfNotAptosFramework` earlier need to be renamed to `AbortsIfNotSupraFramework`

Reply:
Resolved.